src/internal/config/config.go

```diff
@@ -67,6 +68,13 @@ type Service struct {
 	lock *sync.RWMutex
 }
 
+// BackoffCfg represents values used to configure
+// reconnecting hook when http errors occur.
+type BackoffCfg struct {
+	LengthSeconds int `json:"length_seconds"`
```

Comment:
Please use `camelCase`. We try to stick to the standard of the respective format. `camelCase` for JSON, `snake_case` for YAML. There are a lot of occurrences in the code that say the opposite, but they will be fixed.